yarn.lock

```diff
@@ -1406,6 +1406,19 @@
     "@uifabric/set-version" "^7.0.23"
     tslib "^1.10.0"
 
+"@fluentui/make-styles@^0.6.0":
```

Comment:
You should run `yarn syncpack:fix` to fix your local package version references to prevent pulling old versions in.

Reply:
Oh, that's fantastic to know, I was trying to find out what was the script to do that! 😂